Libraries/Components/Picker/Picker.js

```diff
@@ -13,8 +13,9 @@
 'use strict';
 
 var ColorPropType = require('ColorPropType');
-var PickerIOS = require('PickerIOS');
-var PickerAndroid = require('PickerAndroid');
+//var PickerIOS = require('PickerIOS');
```

Comment:
We have to keep everything working on iOS and Android, I don't think we can comment these out.